app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -98,7 +61,7 @@ interface VariantManager {
         )
 
         val REFERRER_VARIANTS = listOf(
-            Variant(RESERVED_EU_AUCTION_VARIANT, features = listOf(SuppressHomeTabWidgetCta), filterBy = { noFilter() })
```

Comment:
@CDRussell I forgot to ping you here as the primary contact for EU auction topics. This has been removed.
Any CTA related to the widget is controlled by the CTA domain logic (e.g: checking if the widget is already added to the home screen. No need to control this from the variant manager)